src/main/java/com/yevhenberladyniuk/debts/service/DebtServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+package com.yevhenberladyniuk.debts.service;
+
+import com.yevhenberladyniuk.debts.domain.Debt;
+import com.yevhenberladyniuk.debts.domain.Partner;
+import com.yevhenberladyniuk.debts.domain.User;
+import com.yevhenberladyniuk.debts.dto.CreateDebt;
+import com.yevhenberladyniuk.debts.dto.DebtDto;
+import com.yevhenberladyniuk.debts.repository.DebtRepository;
+import com.yevhenberladyniuk.debts.repository.PartnerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class DebtServiceImpl implements DebtService{
+
+    private DebtRepository debtRepository;
+
+    private PartnerRepository partnerRepository;
+
+    private  PartnerService partnerService;
+
+    @Autowired
+    public DebtServiceImpl(DebtRepository debtRepository, PartnerRepository partnerRepository, PartnerService partnerService) {
+        this.debtRepository = debtRepository;
+        this.partnerRepository = partnerRepository;
+        this.partnerService = partnerService;
+    }
+
+    @Override
+    public List<Debt> findAllByPartner(Long partnerId, User user) {
+        return debtRepository.findAllByPartnerIdOrderByTransactionDateDesc(partnerId);
```

Comment:
1. Find `partner` by Id
2. Verify that `partner` belongs to `user`, or throw exception like "Not allowed"
3. Find all by partner